### PR TITLE
Add fixture debian-invalid

### PR DESCRIPTION
**Pull request by quba42**
Forms the basis for several new pulp_deb tests we are working on.

Adds invalid Debian repository fixtures.
One release is made invalid by removing a set of Packages files.
Another is made invalid by retroactively altering the Description field in the Release file, and thus invalidating the metadata signature.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,6 +16,10 @@ help:
 	@echo "        Remove fixture data and 'gnupghome'."
 	@echo "    fixtures"
 	@echo "        Create all fixture data."
+	@echo "    fixtures/debian"
+	@echo "        Create Debian repository fixtures."
+	@echo "    fixtures/debian-invalid"
+	@echo "        Create invalid Debian repository fixtrues."
```
Comment (mdellweg) [inline Makefile]:
```suggestion
	@echo "        Create invalid Debian repository fixtures."
```

Comment (quba42) [reply Makefile]: Done.

File: README.rst [hunk 2]
```diff
@@ -50,6 +50,12 @@ See ``make help``.
 ``clean``
     No exotic dependencies are needed.
 
+``fixtures/debian``
+    The ``equivs`` and ``reprepro`` packages must be installed.
```
Comment (mdellweg) [inline README.rst]:
```suggestion
    The ``gnupg``, ``equivs`` and ``reprepro`` packages must be installed.
```

File: README.rst [hunk 3]
```diff
@@ -50,6 +50,12 @@ See ``make help``.
...
+
+``fixtures/debian-invalid``
+    Requires the same dependencies as the ``fixtures/debian`` target.
```
Comment (mdellweg) [inline README.rst]:
```suggestion
    Depends on the ``fixtures/debian`` target.
```
For regular make users, this should imply that fixtures/debian is also built.

File: README.rst (hunk 2 above)
Comment (quba42) [reply README.rst]: Done.

File: README.rst (hunk 3 above)
Comment (quba42) [reply README.rst]: And done.

**Pull request merged**